tests/run_profiling.py

```diff
@@ -259,7 +259,7 @@ def get_header_info(vic_exe, vic_global):
     try:
         header_kwargs['vic_version'] = subprocess.check_output(
             [vic_exe, '-v']).decode()
-    except subprocess.CalledProcessError:
+    except:
```

Comment:
did you mean to make changes to this file?  These all seem unrelated to the Routing Extension.